patches/minecraft/net/minecraft/client/gui/screens/worldselection/WorldSelectionList.java.patch

```diff
@@ -16,6 +16,15 @@
           if (this.f_101693_.f_91066_.m_231828_().m_231551_() || p_283396_) {
              p_281612_.m_280509_(p_282820_, p_283181_, p_282820_ + 32, p_283181_ + 32, -1601138544);
              int j = p_283204_ - p_282820_;
+@@ -483,7 +_,7 @@
+                this.m_170323_();
+             }
+ 
+-            this.f_101693_.m_91152_(this.f_101694_);
++            if(this.f_101693_.f_91080_ instanceof ProgressScreen) this.f_101693_.m_91152_(this.f_101694_); // Neo - fix create world screen being hidden by selection screen
```

Comment:
Please add a space after the `if`. Additionally, the comment should link the Mojira ticket number.

Reply:
We prefer regenerating patches every single time. Surgery on patch files has led to issues in the past. However, we have checks to demonstrate when this issue occurs. But we still prefer proper regeneration :)